config.yml

```diff
@@ -0,0 +1,56 @@
+# GitLab user. git by default
+user: git
+
+# Url to gitlab instance. Used for api calls. Should end with a slash.
+# Default: http://localhost:8080/
+# You only have to change the default if you have configured Unicorn
+# to listen on a custom port, or if you have configured Unicorn to
+# only listen on a Unix domain socket.
+gitlab_url: "http://192.168.59.103:8080/box/api/gls"
+
+http_settings:
+#  user: someone
+#  password: somepass
+#  ca_file: /etc/ssl/cert.pem
+#  ca_path: /etc/pki/tls/certs
+  self_signed_cert: false
+
+# Repositories path
+# Give the canonicalized absolute pathname,
+# REPOS_PATH MUST NOT CONTAIN ANY SYMLINK!!!
+# Check twice that none of the components is a symlink, including "/home".
+repos_path: "/home/git/repositories"
+
+# File used as authorized_keys for gitlab user
+auth_file: "/home/git/.ssh/authorized_keys"
+
+# Redis settings used for pushing commit notices to gitlab
+redis:
+  bin: /usr/bin/redis-cli
+  # host: 127.0.0.1
+  # port: 6379
+  # pass: redispass # Allows you to specify the password for Redis
+  database: 0
+  socket: /var/run/redis/redis.sock # Comment out this line if you want to use TCP
+  namespace: resque:gitlab
+
+rabbit:
+  enabled: true
+  hosts:
+    - 192.168.59.103
```

Comment:
Yes, this should be the service name(s) of the rabbitmq server (or Active MQ server).